src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -538,13 +549,14 @@ private void updateStatisticsOnExpire()
         if (logger.isInfoEnabled())
         {
             StringBuilder sb = new StringBuilder("expire_conf,");
-            sb.append(logPrefix)
-                .append("duration=").append(durationSeconds)
+            sb.append("duration=").append(durationSeconds)
                 .append(",conf_completed=")
                     .append(videobridgeStatistics.totalConferencesCompleted)
                 .append(",has_failed=").append(hasFailed)
                 .append(",has_partially_failed=").append(hasPartiallyFailed);
-            logger.info(Logger.Category.STATISTICS, sb.toString());
+            //TODO(brian): bring back cateogry
```

Comment:
I don't know if the category makes sense anymore. It was briefly used with a fluentd setup a long time ago.

Reply:
so do you think this should be removed completely? or just put on debug?

Reply:
No, I think it's fine in INFO, I'm just saying we can remove the TODO and not worry about it (I don't think the conf_completed part is needed either).